content/docs/introduction/roadmap.md

```diff
@@ -11,13 +11,20 @@ features and current work, see the issue trackers for the various repositories,
 for example, the [Prometheus
 server](https://github.com/prometheus/prometheus/issues).
 
+### New storage engine
+
+A new more efficient storage engine is in development, offering reduced
```

Comment:
Slightly long sentance.  Mabye `A new, more efficient, storage engine is in development.  It will offer reduced...`